Assets/HoloToolkit-Examples/UX/Readme/README_ProgressExample.md

```diff
@@ -0,0 +1,82 @@
+# Progress Example
+<img src="/External/ReadMeImages/MRTK_Progress1.jpg" width="650">
+A progress control provides feedback to the user that a long-running operation is underway. It can mean that the user cannot interact with the app when the progress indicator is visible, and can also indicate how long the wait time might be, depending on the indicator used.
+
+This example scene illustrates the use of the ProgressIndicator prefab to create a progress Indicator with several options for appearance.
+
+## Demo Video
+https://gfycat.com/JaggedDimLacewing
+
+## Types of Progress
+![Progress Types](/External/ReadMeImages/MRTK_Progress2.jpg)
+The ProgressIndicator can appear:
+1. Default orbiting dots animation
+2. A user defined icon which can be static or set to rotate
+3. A user defined prefab which can be static or set to rotate
+4. A user defined text message only such as "Loading..."
+5. Text displayed with a numeric run up of percent loaded
+6. A progress bar
+
+![Progress Type Buttons](/External/ReadMeImages/MRTK_Progress3.jpg)
+
+Different types of Progress control can be opened by pressing these buttons in the scene.
+
+
+
+## ProgressIndicator Prefab
+
+The **ProgressIndicator prefab** is located in HoloToolkit/UX/Prefabs/Progress. It is composed of **ProgressIndicator** script and **Solver** scripts for the billboarding and tag-along behavior.
```

Comment:
Link to HoloToolkit/UX/Prefabs/Progress?

Reply:
Added links